packages/aws-cdk/lib/api/cloudwatch-logs.ts

```diff
@@ -0,0 +1,94 @@
+import * as cxapi from '@aws-cdk/cx-api';
```

Comment:
The `api` folder is getting a little crowded. Let's put this next to the `logs-monitor.ts` file maybe? According to my suggestion above, that would be in `lib/api/logs/`.